from __future__ import absolute_import

from ultron8.exceptions import UltronBaseException


class ValueValidationException(UltronBaseException):
    pass
